src/components/common/Footer/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { Link } from 'react-router-dom';
+import CATEGORY from 'static/category';
+import useMediaQuery from 'utils/hooks/useMediaQuery';
+import styles from './Footer.module.scss';
+
+function Footer(): JSX.Element {
+  const isMobile = useMediaQuery();
+  return (
+    <footer className={styles.Footer}>
+      <div className={styles.Footer__content}>
+        {!isMobile && (
+          <ul className={styles.Footer__services}>
+            {CATEGORY
+              .map((categoryInfo) => categoryInfo.submenu)
+              .flat()
```

Comment:
사소하지만, [flatMap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap) 을 써보는 것은 어떨까요?

```suggestion
              .flatMap(({ submenu }) => submenu)
```